src/Microsoft.Data.SqlClient/src/Microsoft/Data/SqlClient/SqlClientEventSource.Windows.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+
+namespace Microsoft.Data.SqlClient
+{
+    internal partial class SqlClientEventSource : EventSource
+    {
+        private bool _traceLoggingProviderEnabled = false;
+
+        /// <summary>
+        /// Captures application flow traces from native networking implementation
+        /// </summary>
+        private const EventCommand SNINativeTrace = (EventCommand)8192;
+
+        /// <summary>
+        /// Captures scope trace events from native networking implementation
+        /// </summary>
+        private const EventCommand SNINativeScope = (EventCommand)16384;
+
+        /// <summary>
+        /// Disables all event tracing in native networking implementation
+        /// </summary>
+        private const EventCommand SNINativeDisable = (EventCommand)32768;
+
+        protected override void OnEventCommand(EventCommandEventArgs e)
+        {
+            // Internally, EventSource.EnableEvents sends an event command, with a reserved value of 0, -2, or -3.
```

Comment:
I don't see EventSource has EnableEvents method. Do you mean EventListener.EnableEvents?

Reply:
Yes, fixed.